edu.princeton.physics.WSJTX.metainfo.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>edu.princeton.physics.WSJTX.desktop</id>
```

Comment:
```suggestion
  <id>edu.princeton.physics.WSJTX</id>
```

Reply:
Hmm, yeah. The desktop suffix is not needed.

Reply:
Note it's better to add `<launchable type="desktop-id">edu.princeton.physics.WSJTX.desktop</launchable>` if you drop desktop suffix.

Reply:
It should not be needed if the desktop file name matches the app id.